ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -275,12 +280,17 @@ void RSkComponentTextInput::processEventKey (rnsKey eventKeyType,bool* stopPropa
     //is always 0 & selectionRange.location always end 
     textInputMetrics.selectionRange.location = cursor_.end ;
     textInputMetrics.selectionRange.length = 0;
+    int newLength = textString.length();
```

Comment:
can we name these variables like textLengthBeforeEdit and textLengthAfterEdit ; to keep it meaningful